src/guides/v2.3/frontend-dev-guide/css-guide/css_quick_guide_approach.md

```diff
@@ -26,6 +26,31 @@ To extend the parent theme's styles in your theme:
 
 1. Add your Less code in this file.
 
+However, the above only works if the theme's parent is a Blank. Consider a theme Theme A which is the child of Blank. Theme A has two children, B and C. A global style rule is added to the `_extend.less` file of theme A. This extends its parent Blank. Theme B and C also have their own `_extend.less` files. Now, Theme B and C will override the parent's (Theme A), rather than extending it further. So, Theme B & Theme C are extending their grandparent (Blank) and overriding their parent (Theme A) in this setup.
```

Comment:
```suggestion
However, the above only works if the theme's parent is a Blank. Consider a Theme A which is the child of Blank. Theme A has two children, B and C. A global style rule is added to the `_extend.less` file of theme A. This extends its parent Blank. Theme B and C also have their own `_extend.less` files. Theme B and C will override the parent (Theme A), rather than extending it further. Theme B & Theme C are extending their grandparent (Blank) and overriding their parent (Theme A) in this setup.
```